pkg/telemetry/cassandra_metrics_filters.go

```diff
@@ -0,0 +1,44 @@
+package telemetry
+
+import (
+	"strings"
+
+	telemetry "github.com/k8ssandra/k8ssandra-operator/apis/telemetry/v1alpha1"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/cassandra"
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	DefaultFilters = []string{"deny:org.apache.cassandra.metrics.Table",
```

Comment:
These should be documented, maybe both in telemetry_types.go as well as at docs.k8ssandra.io.